docs/user-guide/configuring.md

```diff
@@ -550,84 +550,125 @@ The complete configuration hierarchy, from highest precedence to lowest preceden
 
 ## Extending Configuration Files
 
-If you want to extend a specific configuration file, you can use the `extends` property and specify the path to the file. The path can be either relative or absolute.
+A configuration file can "extend" one or more "child" configurations to override a "base" set of rules.
```

Comment:
Why the quotes around these terms? If the intent is to introduce new terms, then I'd suggest formally defining them in a terminology section.